python/Ganga/__init__.py

```diff
@@ -26,6 +26,14 @@ def getLCGRootPath():
     else:
         return ''
 
+def getDefaultUserName():
+    # detect default user (equal to unix user name)
+    import getpass
+    try:
+        return getpass.getuser()
+    except Exception as x:
+        raise Ganga.Utility.Config.ConfigError('Cannot get default user name' + str(x))
```

Comment:
If we never handle this exception (and I'm not sure under what circumstances `getuser()` would raise), perhaps we can make the line below be just:
```python
conf_config.addOption('user', getpass.getuser(), ...
```